### PR TITLE
Multiple RUN, ADD, COPY, Instructions Listed query

### LINKED ISSUE #2370: Add Multiple RUN, ADD, COPY, Instructions Listed query for Dockerfile
### Platform
Dockerfile

### Description
You should group multiple commands (RUN, Copy, And) in order to reduce the number of layers.

### PR DESCRIPTION
**Pull request by fabioGoncalvesCx**
Closes #2370

**Proposed Changes**

- Group multiple commands (RUN, Copy, And) in order to reduce the number of layers.

I submit this contribution under Apache-2.0 license.

### CODE REVIEW

File: assets/queries/dockerfile/multiple_run_add_copy_instructions_listed/query.rego [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	lineCounter := [x | cmdInst[n].StartLine - cmdInst[m].StartLine == -1; x := cmdInst[n]]
+
+	upperName := upper(instruction)
+	countCmdInst := count(lineCounter)
+	countCmdInst > 0
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("FROM={{%s}}.{{%s}}", [name, lineCounter[0].Original]),
+		"issueType": "Best Practices",
```
Comment (ruigomescx) [inline assets/queries/dockerfile/multiple_run_add_copy_instructions_listed/query.rego:20]: issueType is wrong: We could use "Redundant Attribute"

File: assets/queries/dockerfile/multiple_run_add_copy_instructions_listed/query.rego [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		"keyExpectedValue": sprintf("There isn´t any %s instruction that could be grouped", [upperName]),
+		"keyActualValue": sprintf("There are %s instructions that could be grouped", [upperName]),
+	}
+}
```
Comment (ruigomescx) [inline assets/queries/dockerfile/multiple_run_add_copy_instructions_listed/query.rego:70]: This code is repeated, with only the instruction to check changing. Can we refactor this to just one CxPolicy?

**Review by ruigomescx — CHANGES_REQUESTED** [review]

File: assets/queries/dockerfile/multiple_run_add_copy_instructions_listed/query.rego (hunk 2 above)
Comment (fabioGoncalvesCx) [reply assets/queries/dockerfile/multiple_run_add_copy_instructions_listed/query.rego]: Yes, Done!

**Review by rogeriopeixotocx — CHANGES_REQUESTED** [review]
add more samples to cover ADD and COPY scenarios.

separate them by different files:
```
positive1.tf
postive2.tf
negative1.tf
and so on...
```

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**